formats/shared/definitions/publishing_api_base.jsonnet

```diff
@@ -15,6 +15,12 @@
           type: "string",
         },
       },
+      organisations: {
+        type: "array",
+        items: {
+          type: "string",
+        },
+      },
```

Comment:
This should be a `"$ref": "#/definitions/guid_list"` like auth_bypass_ids. It'd also be useful to have a description and have that state that this is organisation content ids.